ci/prepare-tasks-for-signing.js

```diff
@@ -0,0 +1,35 @@
+var fs = require('fs');
+var path = require('path');
+var util = require('./ci-util');
+
+var buildTasksPath = path.join(__dirname, '..', '_build', 'Tasks');
+var signingPath = path.join(__dirname, '..', '_build', '_signedTasks');
+
+// For each task that changed
```

Comment:
I wonder if we want to move this to `make.js` then you can use `taskList` and we can have a command like `npm run prepareTasksForSigning` that we can call in CI?

Reply:
Let's discuss that in the meeting. I don't know why things are the way they are currently. I.e. somethings done in Powershell, some in node, some in yaml, some in the make.js. It's all over the place.